src/api-reference/request/v4.endpoints.travel-agency-resources.markdown

```diff
@@ -0,0 +1,77 @@
+---
+title: Travel Request v4 - Travel Agency Resources
+layout: reference
+---
+
+{% include prerelease.html %}
+
+* [Travel Agency Resources](#travel-agency-resource)
+  * [Get the description of a travel Agency](#get-travel-agency-description-resource)
+
+## HTTP Status Codes
+
+To learn more about response HTTP status codes for this API see [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html).
+
+# <a name="travel-agency-resource"></a>Travel Agency Resources
+
+Manage the configuration for Travel Agencies integrated with Concur Request.
+
+## <a name="get-travel-agency-description-resource"></a>Get the description of a Travel Agency
+
+##### URI Template
+
+```
+GET {datacenter}/travelrequest/v4/travelagencies/{agencyUuid}
+```
+
+### Scopes
+
+`travelrequest.write` - Refer to [Scope Usage](./v4.get-started.html#scope-usage) for full details.
+
+### HTTP Request
+
+#### Parameters
+
+Name|Parameter Type|Data Type|Description
+---|---|---|---
+agencyUuid|path|string|**Required** The unique identifier of the Travel Agency
+
+#### Payload
+
+None
+
+### HTTP Response
+
+#### Payload
+
+Name|Type|Format|Description
```

Comment:
This schema should be on the consolidated Schemas page.

Reply:
OK this was the case before this is why we did not mage any changes
@laurentlhermine how can I provide a new file with the following content:
* update the schema file by adding a Travel agency resource 
* and make this payload reference to this new resource?

Reply:
OK
@laurentlhermine  I have modified the v4_Request_endpoint schemas_v8 file and v4_Request_travel agency resources_v8 files accordingly, available in https://jira.concur.com/browse/TVR-10569